packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx

```diff
@@ -1,146 +1,129 @@
 import { crabV2graphOptions } from '@constants/diagram'
+import HighchartsReact from 'highcharts-react-official'
+import Highcharts from 'highcharts'
 
 
 export type ChartDataInfo = {
     timestamp: number
     crabPnL: number
+    crabEth: number
+    crabUsd: number
+    ethUsd: number
   }
 
-
-enum ChartType {
-    PNL = 'PNL',
-}
-const chartTradeTypeAtom = atom(0)
-
 const useStyles = makeStyles((theme) =>
   createStyles({
     container: {
- 
       padding: theme.spacing(0),
       marginTop: theme.spacing(4),
       marginBottom: theme.spacing(1),
       maxWidth: '640px',
     },
-    green: {
-      color: theme.palette.success.main,
-    },
-    red: {
-      color: theme.palette.error.main,
-    },
-    navDiv: {
-        display: 'flex',
-        marginBottom: theme.spacing(2),
-        alignItems: 'center',
-        marginTop: theme.spacing(2),
+    chartContainer: {
+      borderRadius: theme.spacing(1),
+      padding: theme.spacing(1, 2),
+      display: 'flex',
+      marginTop: theme.spacing(2),
+      [theme.breakpoints.up('sm')]: {
+        maxHeight: '310px',
       },
-    legendBox: {
-        display: 'flex',
-        gap: '10px',
-        marginTop: '10px',
-        justifyContent: 'center',
+      [theme.breakpoints.down('sm')]: {
+        flexDirection: 'column',
       },
-      payoffContainer: {
-        //  background: theme.palette.background.stone,
-         borderRadius: theme.spacing(1),
-        padding: theme.spacing(1, 2),
-        display: 'flex',
-        marginTop: theme.spacing(2),
-        [theme.breakpoints.up('sm')]: {
-          maxHeight: '310px',
-        },
-        [theme.breakpoints.down('sm')]: {
-          flexDirection: 'column',
-        },
        
-      },
+    },
   }),
 )
-const Chart = dynamic(() => import('kaktana-react-lightweight-charts'), { ssr: false })
-
-const modeAtom = atom<ChartType>((get) => {
-    const tradeType = get(chartTradeTypeAtom)
-    if (tradeType === 0) return ChartType.PNL
-    return ChartType.PNL
-})
 
 function StrategyPnLV2() {
     
     const classes = useStyles()
-    const mode = useAtomValue(modeAtom)
     const query = useCrabPnLV2ChartData()
-
  
-    const pnlSeries = query?.data?.data.map((x: ChartDataInfo) => ({ time: x.timestamp, value:x.crabPnL*100 })) ;
-    const zeroSeries = [{ price: 0, color: '#9dbdba' }]
+    const pnlSeries = query?.data?.data.map((x: ChartDataInfo) => ([ x.timestamp*1000, x.crabPnL*100 ])) ;
+    const crabEthSeries = query?.data?.data.map((x: ChartDataInfo) => ([ x.timestamp*1000, x.crabEth ])) ;
+    const crabUsdSeries = query?.data?.data.map((x: ChartDataInfo) => ([ x.timestamp*1000, x.crabUsd ])) ;
+    const ethUsdSeries = query?.data?.data.map((x: ChartDataInfo) => ([ x.timestamp*1000, x.ethUsd ])) ;
+
+    useEffect(() => {
+      Highcharts.setOptions({
+       lang: {
+        thousandsSep: ','
+       }
+      });
+     }, []);
+
+    const series = [{
+      name: 'PnL',
```

Comment:
we can remove this once we make the below 3 % returns, so the graph should only have 3 lines